dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt

```diff
@@ -593,8 +591,8 @@ fun ChutneyStepBuilder.ScpUploadTask(
             "destination" to destination,
             "timeout" to timeout
         ).notEmptyToMap(),
-        outputs = outputs,
-        validations = validations
+        outputs = emptyMap(),
```

Comment:
Since these tasks don't return anything, and the apache mina API does nothing more relevant, I thought that removing outputs and validations from the DSL would avoid any misinterpretation or misused